src/App/Resources/AppResources.resx

```diff
@@ -2214,4 +2214,10 @@
   <data name="SpecialCharacters" xml:space="preserve">
     <value>Special Characters (!@#$%^&amp;*)</value>
   </data>
+  <data name="PasswordIsVisible" xml:space="preserve">
+    <value>Password is visible, tap to hide.</value>
+  </data>
+  <data name="PasswordIsNotVisible" xml:space="preserve">
+    <value>Password is not visible, tap to show.</value>
+  </data>
```

Comment:
IMO the key and the value should be the same here: `PasswordIsVisibleTapToHide` (same for the other one)